var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -25,10 +25,11 @@ class Abinit(AutotoolsPackage):
     """
 
     homepage = "https://www.abinit.org/"
```

Comment:
```suggestion
    homepage = "https://abinit.github.io/abinit_web/"
```
Since https://www.ssllabs.com/ssltest/analyze.html?d=www.abinit.org&latest reports that the certificate chain reported by www.abinit.org is incomplete and doesn't include intermediate certificates. This requires the browser to rely on what has been cached already, and users who have not happened upon a website with the intermediate certificate will not be able to access.

See also e.g.
```console
$ curl -L https://www.abinit.org/
curl: (35) OpenSSL/3.3.1: error:80000002:system library::No such file or directory
```

Reply:
Is applied.